python/py-emcee3/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-emcee3
+version             3.0.2
+distname            emcee-${version}
+master_sites        pypi:[string index ${python.rootname} 0]/[string range ${python.rootname} 0 end-1]
```

Comment:
typically you don't need to set `distname` and `master_sites` these are the default already set by the python PG. Since here the name is `emcee3` is different than the one on PyPI (i.e., `emcee`), you should instead add `python.rootname emcee`; then you can remove the two fields mentioned above and also the livecheck will work properly.

Reply:
Thanks! Done.